pkg/storage/storage.go

```diff
@@ -122,7 +122,10 @@ func (s *Storage) Deployed(name string) (*rspb.Release, error) {
 		return nil, errors.Errorf("%q has no deployed releases", name)
 	}
 
-	return ls[0], err
+	// We want oldest to newest
+	relutil.SortByRevision(ls)
```

Comment:
usually the order is correct without sort. may be some storage driver need sort.